popgen/dunn.go

```diff
@@ -64,18 +61,14 @@ func FindMinInter(g []*Group, subFa []fasta.Fasta) int {
 	var answer int = math.MaxInt64
 	faMap := fasta.ToMap(subFa)
 	var faI, faJ []dna.Base
-	var faFound bool
+	var faFoundI, faFoundJ bool
 	for i := 0; i < len(g[0].Members); i++ {
 		for j := 0; j < len(g[1].Members); j++ {
-			faI, faFound = faMap[g[0].Members[i]]
-			if !faFound {
-				log.Panicf("%s not found in fasta map", g[0].Members[i])
-			}
-			faJ, faFound = faMap[g[1].Members[j]]
-			if !faFound {
-				log.Panicf("%s not found in fasta map", g[1].Members[j])
+			faI, faFoundI = faMap[g[0].Members[i]]
+			faJ, faFoundJ = faMap[g[1].Members[j]]
+			if faFoundI && faFoundJ {
```

Comment:
Similar comment to above.  Seems like it should be an error if these seq are not found.  I worry about asking for the Dunn's for a group of 10 against a group of 5, and it silently calcs a group fo 8 against a group of 4.